### PR TITLE
LPS-121343 Enable tests of deleteField and remove useless test

### PR DESCRIPTION
**Pull request by alinedoleron**
Related Issue: https://issues.liferay.com/browse/LPS-121343

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @alinedoleron, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldDeletedHandler.es.js [hunk 1]
```diff
@@ -15,69 +15,30 @@
...
+				fieldName: 'radio',
+				removeEmptyRows: true,
 			};
 			const state = {
 				pages: mockPages,
 				rules: [],
 			};
 
-			const removeRowSpy = jest.spyOn(FormSupport, 'removeRow');
+			const removeRowSpy = jest.spyOn(FormSupport, 'removeEmptyRows');
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldDeletedHandler.es.js:33]: @alinedoleron, please name the variable consistently, e.g. rename it to `removeEmptyRowsSpy`. Thanks.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldDeletedHandler.es.js [hunk 2]
```diff
@@ -15,69 +15,30 @@
...
 
-			fieldDeletedHandler.handleFieldDeleted(state, event);
+			fieldDeletedHandler.handleFieldDeleted({}, state, event);
 
 			expect(removeRowSpy).toHaveBeenCalled();
 
 			removeRowSpy.mockRestore();
 		});
-
-		it('calls clearAllConditionFieldValues() when deleting a field used as the first operand of a condition', () => {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldDeletedHandler.es.js:45]: @alinedoleron, why this test is no longer necessary? Can you please clarify?

Comment (alinedoleron) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldDeletedHandler.es.js:45]: Hi @natocesarrego, it was a mistake. I think I may be debug wrongly. But I'm still having problem with this test.
It says it failed but actually the method is being called if I debug it.

```
LayoutProvider/handlers/fieldDeletedHandler › handleFieldDeleted(state, event) › calls clearAllConditionFieldValues() when deleting a field used as the first operand of a condition

    expect(jest.fn()).toHaveBeenCalled()

    Expected number of calls: >= 1
    Received number of calls:    0

      78 | 			fieldDeletedHandler.handleFieldDeleted({}, state, event);
      79 |
    > 80 | 			expect(clearAllConditionFieldValuesSpy).toHaveBeenCalled();
         | 			                                        ^
      81 |
      82 | 			clearAllConditionFieldValuesSpy.mockRestore();
      83 | 		});

      at Object.toHaveBeenCalled (test/js/components/LayoutProvider/handlers/fieldDeletedHandler.es.js:80:44)


  console.log src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js:52
    entrou!!!
```

And this 'entrou!!!' I put into the method `clearAllConditionFieldValuesSpy`. =/ I don't understand the reason it is happening.

Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldDeletedHandler.es.js:45]: @alinedoleron, let's screen share after our daily meeting to talk about this pull. Thanks.

**Comment by alinedoleron** [comment]
new pr: https://github.com/natocesarrego/liferay-portal/pull/867

**Pull request closed without merging**